lib/stack_master/commands/apply.rb

```diff
@@ -162,6 +164,13 @@ def ensure_valid_template_body_size!
         end
       end
 
+      def offer_to_run_dependant_stacks
+        @dependancies.outdated_stacks.each do |stack|
+          next unless ask?(%Q{We found a dependent stack "#{stack.stack_name}" which is now out of date because of this change.\nWould you like us to apply this stack now (y/n)?})
```

Comment:
For consistency with other prompts in `stack_master`, consider leaving out the I/We and using a more direct tone.

Compare with prompt to apply a change set:

"Apply change set (y/n)? "